src/guides/v2.3/javascript-dev-guide/javascript/js_init.md

```diff
@@ -133,6 +133,9 @@ require([
 </script>
 ```
 
+{:.bs-callout-tip}
+For control when scripts is executed we recommend all developers should use declarative syntax over imperative. When use imperative syntax sometimes you do not have the ability to leverage any existing JS classes, and it can block rendering of the page.
```

Comment:
```suggestion
For better control when scripts are executed, it is recommended that developers use the declarative syntax over the imperative. When using imperative syntax, it can mean that the ability to leverage any existing JS classes is lost and it can block the rendering of the page.
```
Does this still read as correct, informatively speaking?